litgpt/data/prepare_slimpajama.py

```diff
@@ -2,19 +2,12 @@
 
 import json
 import os
-import sys
 import time
 from pathlib import Path
 
-import zstandard as zstd
-from lightning.data.streaming import DataChunkRecipe, DataProcessor
-
-# support running without installing as a package
-wd = Path(__file__).parent.parent.resolve()
-sys.path.append(str(wd))
-
 from litgpt import Tokenizer
 from litgpt.utils import CLI
+from litgpt.data.prepare_starcoder import DataChunkRecipe
```

Comment:
I don't get this part. 
Since we are using the latest lightning commit and the [data] extra, this should already use lit_data.

But ok, I can look at this again in #1034

Reply:
This is for the base installation (not requirements-all) which doesn't include data